source/connect/advanced-connection-options.txt

```diff
@@ -0,0 +1,46 @@
+.. _advanced-connection-options:
+
+===========================
+Advanced Connection Options
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/intro-advanced-connection-options.rst
+
+Options
+-------
+
```

Comment:
(proper noun?)
Just a question. Is Connection String Scheme truly a proper noun?

Reply:
Looks like it was mistakenly capitalized bc of the styling in the Compass UI. Fixing across docs.